crates/schema/src/def/validate/v9.rs

```diff
@@ -755,11 +769,73 @@ enum TypeDefOrUse {
     Use,
 }
 
+fn check_table_names_consistent_with_type_names(
```

Comment:
Why do we have this requirement? Can I not have this:

```rust
#[table(name="logged_in_user")]
#[table(name="logged_out_user")]
struct User {
    ...
}
```

Reply:
oh you're completely right, i was just pondering if I actually needed this. I will discard.